### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by elias-knodel**
## Description  
Adds better context to the README.md

## Related issue  
#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,82 @@
...
+  [![pnpm-version][pnpm-version]][pnpm-url]
+
+  [![commit-info][commit-info]][commit-url]
+  [![contributors-info][contributors-info]][contributors-url]
+  [![repo-size-info][repo-size-info]][repo-size-url]
+
+  <br><hr><br>
+
+# DeepCity CLI
+~ This CLI serves as an interface between your commands and our alias ~
```
Comment (pascal-zarrad) [inline README.md]: This phrase sounds pretty weird, we should change it.

Comment (elias-knodel) [reply README.md]: yes my lord

Comment (elias-knodel) [reply README.md]: Yeah... i had no ideas for a description back then ^^"

Comment (pascal-zarrad) [reply README.md]: Do you have now an idea?

**Pull request merged**